### DISCUSSION TITLE
App Store Connect API private key leaking into logs

### DISCUSSION THREAD

**Discussion by carlo-** [fastlane/fastlane#18125] [Ideas] [answered]
~The full App Store Connect API private key is currently leaking into the log when running Fastlane, which is a security concern for some CI setups, and seems unnecessary. Perhaps this could be avoided completely, or maybe the key should only be shown if `--verbose` is used, or similar. Thoughts?~
Edit: this no longer happens after v. `2.161.0`.
 
<img width="1133" alt="Screen Shot 2021-02-04 at 14 00 45" src="https://user-images.githubusercontent.com/7290609/106896514-f0522980-66f1-11eb-9a48-8dbfe5bb948f.png">

**Comment by dinsen** [comment]
> The full App Store Connect API private key is currently leaking into the log when running Fastlane, which is a security concern for some CI setups, and seems unnecessary. Perhaps this could be avoided completely, or maybe the key should only be shown if `--verbose` is used, or similar. Thoughts?
> 
>  
> 
> <img width="1133" alt="Screen Shot 2021-02-04 at 14 00 45" src="https://user-images.githubusercontent.com/7290609/106896514-f0522980-66f1-11eb-9a48-8dbfe5bb948f.png">
> 

Hi

When is this exposed?

**Reply by carlo-** [reply]
This was shown in the lane context if an error occurred after logging in with `app_store_connect_api_key`, but as mentioned in the other thread, it has been fixed in v. `2.161.0`.

**Comment by joshdholtz** [comment, marked as answer]
@carlo- Which version of _fastlane_ are you on? I patched this in `2.161.0` so it shouldn’t be appearing in that table 🤔

**Reply by carlo-** [reply]
Indeed it seems that this has been fixed! I was running an older version when it happened. Sorry for the false alarm 😅

**Reply by joshdholtz** [reply]
No problem at all! I'm glad that you were using an older version otherwise I would have been confused on what was going on 😇
